db/tag.c

```diff
@@ -301,7 +301,7 @@ int client_type_to_server_type(int type)
     case CLIENT_VUTF8:
         return SERVER_VUTF8;
     default:
-        return type;
+        abort();
```

Comment:
Maybe assert() that this isn't reachable instead?

Reply:
Asserts aren't enabled for normal builds afaik -- only enabled for DEBUG builds.